core/src/main/scala/org/apache/spark/scheduler/Pool.scala

```diff
@@ -37,25 +37,22 @@ private[spark] class Pool(
 
   val schedulableQueue = new ConcurrentLinkedQueue[Schedulable]
   val schedulableNameToSchedulable = new ConcurrentHashMap[String, Schedulable]
-  var weight = initWeight
-  var minShare = initMinShare
+  val weight = initWeight
```

Comment:
`Pool` extends `Schedulable` trait and needs to override `weight`. It is also used for  `taskToWeightRatio` calculation at `FairSchedulingAlgorithm` level.